docs/design/3_1_2_platform_management.tex

```diff
@@ -24,18 +24,24 @@ \subsubsection{Device Selection}
 \end{itemize} 
 
 \subsubsection{Data Flow}
-The overall data flow managed by the Raspberry Pi is as follows:
+\label{pmb_dataflow}
+The overall data flow managed by the Platform Management Board is as follows:
 \begin{enumerate}
-\item Video is acquired through the camera CSI interface
-\item Frame manipulation/correction is performed on the video, if required by the specific machine learning use-case
-\item Manipulated video is decomposed and transmitted over Ethernet to the programmable logic board (which performs the machine learning)
+\item A video frame ($640px \times 480px$) is acquired through the camera CSI interface
+\item Frame manipulation/correction is performed on the frame, if required by the specific machine learning use-case. For the YOLOv2 accelerator (described in Section \ref{ml_accel}), images sent to the Zedboard are letterboxed to $416px \times 416px$ to adhere to ML IP block restrictions.
+\item The letterboxed frame is decomposed and transmitted over Ethernet to the programmable logic board (which performs the machine learning)
+\begin{enumerate}
+\item While waiting for the machine learning results, the PMB sends full-resolution ($640px \times 480px$) frames to the Base Station -- allowing for higher "background" framerates rather than tying framerates to ML results.
```

Comment:
```suggestion
\item While waiting for the machine learning results, the PMB sends full-resolution ($640px \times 480px$) frames to the Base Station --- allowing for higher ``background" framerate rather than tying framerate to ML results.
```